utilities/ssh.py

```
@@ -0,0 +1,614 @@
+            stdout=stdout,
+            stderr=stderr,
+        ) from exc
+    except OSError as exc:
+        raise SSHConnectionError(f"Failed to execute virtctl ssh: {exc}") from exc
+
+
+def run_ssh_commands(
+    vm: VirtualMachineForTests,
+    commands: list[str],
+    timeout: int | float = TIMEOUT_1MIN,
+    check: bool = True,
```

Ruff (0.14.14)

Boolean-typed positional argument in function definition

(FBT001)